aiohttp/client_reqrep.py

```diff
@@ -291,19 +290,20 @@ def __init__(
     ):
         if loop is None:
             loop = asyncio.get_event_loop()
-
-        match = _CONTAINS_CONTROL_CHAR_RE.search(method)
-        if match:
+        if match := _CONTAINS_CONTROL_CHAR_RE.search(method):
             raise ValueError(
                 f"Method cannot contain non-token characters {method!r} "
-                "(found at least {match.group()!r})"
+                f"(found at least {match.group()!r})"
```

Comment:
This was missing an `f-`. It looks like an earlier backport conflict resolution problem